README.md

```diff
@@ -10,101 +9,107 @@
+totalAmount | number | no | | you can pass dynamic amount. dynamic pricing should be checked for this. |
+| supportedCurrencies | string array | no | | currencies you want to support.
 
 ### 2. Dynamic payment with embedded button
 
 Heliopay button also supports dynamic payment options where you can use embedded button in carting for example.
 In the example below if you provide totalAmount and the currency to the button, the user will perform the payments with those specifications.
+
 ```ts
-import { HelioPay } from '@heliofi/react';
+import { HelioPay } from "@heliofi/react";
 
 const App = () => {
   return (
     <div>
       <HelioPay
         cluster="devnet"
-          paymentRequestId={"your_paylink_id"}
-          supportedCurrencies={["SOL"]}
-          totalAmount={0.01}
-          onSuccess={(event: SuccessPaymentEvent): void => {
-            console.log("onSuccess", event);
-            /**
-             * The success event signature looks as follows:
-             * {
-             *    content: string;
-             *    transaction: string;
-             * }
-             * The transaction can be used to verify the payment with helio api
-             * */
-          }}
+        paymentRequestId={"your_paylink_id"}
+        supportedCurrencies={["SOL"]}
+        totalAmount={0.01}
+        onSuccess={(event: SuccessPaymentEvent): void => {
+          console.log("onSuccess", event);
+          /**
+           * The success event signature looks as follows:
+           * {
+           *    content: string;
+           *    transaction: string;
+           * }
+           * The transaction can be used to verify the payment with helio api
+           * */
+        }}
       />
-    </div>)
-}
+    </div>
+  );
+};
 ```
+
 You can verify the payment using helio api for development and for production accordingly:
-`https://test.api.hel.io` and `https://prod.api.hel.io`.
+`https://dev.api.hel.io` and `https://api.hel.io`.
 
 In order to validate the payment you need to register your public key with Helio team, and you will receive api secret token.
 After which call the endpoint like in the example below:
 
 ```ts
 try {
-  const transactionSignature = 'solana blockchain transaction signature';
-  const token = 'token from helio team';
-  const publicKey = 'you public key registered with helio team that recives transacitons';
-  
-  const baseUrl = 'https://test.api.hel.io';
-  const endpoint = `/payment-state/signature/${transactionSignature}?publicKey=${publicKey}`
-  
+  const transactionSignature = "solana blockchain transaction signature";
+  const token = "token from helio team";
+  const publicKey =
+    "you public key registered with helio team that recives transacitons";
+
+  const baseUrl = "https://dev.api.hel.io";
+  const endpoint = `/payment-state/signature/${transactionSignature}?publicKey=${publicKey}`;
```

Comment:
we support both legacy depricated payment-state endpoint but the new one is called `/v1/transactions/signature/{signature}?publicKey=${publickey}`